### DISCUSSION TITLE
some mods reqier this mod are now not working

### DISCUSSION THREAD

**Discussion by Fartonice-Dev** [Arkensor/DayZ-CommunityFramework#120] [General]
Today after logging in I've noticed all the safe zones that I've made with the buildings from your pack are not there....(?) is there update coming or are you aware of this I don't know if it's your mod or maybe a dependency that you need BF Maybe but there's no updates out and you're not the only pack that I'm having a problem with. I've also log into offline mode load up my saved buildings in end zones and they're not working there either

**Comment by Slava20002** [comment]
здраствуйте, мод CF, требуемый для большенсва мои модов на сервере хост, было обновление дейз, но когда прописую в запуск сервера этот мод, то сервер не запускается, почеуму?
